src/actions/actions.js

```diff
@@ -1,4 +1,5 @@
 export const RECEIVE_FORECAST = 'RECEIVE_FORECAST';
+export const TEST_ACTION = 'TEST_ACTION';
```

Comment:
(Part Two)

You made these two, but you didn't make `SELECT_LOCATION` or `INVALIDATE_LOCATION`, so this code will blow up once you start using the container instead of the component as I point out in another file.

Let's fix this:

```js
export const SELECT_LOCATION = 'SELECT_LOCATION';
export const INVALIDATE_LOCATION = 'INVALIDATE_LOCATION';
```